README.md

```diff
@@ -145,7 +145,6 @@ Chai offers a robust Plugin architecture for extending Chai's assertions and int
 - [chaijs / chai-docs](https://github.com/chaijs/chai-docs): The chaijs.com website source code.
 - [chaijs / assertion-error](https://github.com/chaijs/assertion-error): Custom `Error` constructor thrown upon an assertion failing.
 - [chaijs / deep-eql](https://github.com/chaijs/deep-eql): Improved deep equality testing for Node.js and the browser.
-- [chaijs / type-detect](https://github.com/chaijs/type-detect): Improved typeof detection for Node.js and the browser.
```

Comment:
We might consider porting all of the tests from `type-detect` to make sure it's stable. This will also help us highlight what a migration path would look like for the `.as()` assertions to just check `[Symbol.toStringTag]`.

Reply:
Just did, let me know what you think!